sphinx_gallery/binder.py

```diff
@@ -95,17 +118,61 @@ def gen_binder_rst(fname, binder_conf):
     return rst
 
 
-def copy_binder_reqs(app):
+def copy_binder_files(app, exception):
+    """Copy all Binder requirements and notebooks files."""
+    if exception is not None:
+        return
+
+    if app.builder.name not in ['html', 'readthedocs']:
+        return
+
+    gallery_conf = app.config.sphinx_gallery_conf
+    binder_conf = check_binder_conf(gallery_conf.get('binder'))
+
+    # Copy the requirements files for binder
+    if not len(binder_conf) > 0:
+        return
+
+    logger.info('copying binder requirements...', color='white')
+    _copy_binder_reqs(app, gallery_conf, binder_conf)
+    logger.info('copying binder notebooks...', color='white')
+    _copy_binder_notebooks(app, gallery_conf, binder_conf)
+
+
+def _copy_binder_reqs(app, gallery_conf, binder_conf):
```

Comment:
Why is there a need to change this function? `gallery_conf` is not used, remove it. Also use `shutil.copy` not `sh.copy`

Reply:
removed the gallery conf (was from an earlier version of the PR I think). I modified the function so that it has the same general structure as the "copy notebooks" function